recipes/e3sm_to_cmip/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "e3sm_to_cmip" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a23d7440bab9950f6fef8a6e5fe60741e4e5be2a2421d37ca9a1739dab1deef1
+
+build:
+  skip: True  # [win or py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - nco
+    - cmor >=3.5.0
+    - cdutil
+    - cdms2 >=3.1
+    - progressbar2
+    - pathos
+    - pyyaml
+    - xarray
+    - netcdf4
+    - dask
+    - scipy
+
+test:
+  commands:
+    - e3sm_to_cmip --help
+  imports:
+    - e3sm_to_cmip
```

Comment:
@sterlingbaldwin, any other sanity checks you would add?

Reply:
Note: we can always add more checks later once the feedstock is built.

Reply:
This is fine for the time being, running with the "--help" finds most of the issues when dependencies dont load properly.